website/i18n/en.json

```diff
@@ -83,6 +83,32 @@
       },
       "editors/vscode": {
         "title": "Visual Studio Code"
+      },
+      "integrations/new-build-tool": {
+        "title": "Integrating a new build tool"
+      },
+      "integrations/new-editor": {
+        "title": "Integrating a new editor"
+      },
+      "integrations/remote-language-server": {
+        "title": "Remote Language Servers"
+      },
+      "lsp-extensions/debug-adapter-protocol": {
```

Comment:
I think lsp-extensions and integrations are pretty similar. Integrating a new editor for example might need to implement some of the extensions. I would keep it all under `integrations`

Reply:
Sounds good. I moved them all under integrations.